public/js/project_upload.js

```diff
@@ -0,0 +1,52 @@
+$.noConflict();
+
+jQuery(document).ready(function($) {
+
+	$('#myUpload').on('shown.bs.modal', function () {
+
+		$('#uploadSubmit').prop('disabled', true);
+
+	});
+
+	$("#verifyUrl").click(function() {
+
+    	$.ajax({
+
+    		type : 	"post",
+
+    		url : 	$("#uploadForm").prop('action'),
+
+    		data : 	$("#uploadForm").serialize(),
+
+    		success : 	function(data, status, xhr) {
```

Comment:
'xhr' is defined but never used.<br>'status' is defined but never used.<br>'data' is defined but never used.